simulation/src/test/java/com/palantir/dialogue/core/SimulationTest.java

```diff
@@ -198,26 +198,30 @@ public void slow_503s_then_revert() {
     }
 
     @Test
-    public void fast_500s_then_revert() {
-        int capacity = 60;
+    public void fast_503s_then_revert() {
         servers = servers(
                 SimulationServer.builder()
-                        .serverName("fast")
+                        .serverName("server1")
                         .simulation(simulation)
-                        .handler(h -> h.response(200).linearResponseTime(Duration.ofMillis(60), capacity))
+                        .handler(h -> h.response(200).responseTime(Duration.ofMillis(120)))
                         .build(),
                 SimulationServer.builder()
-                        .serverName("fast_500s_then_revert")
+                        .serverName("server2")
                         .simulation(simulation)
-                        .handler(h -> h.response(200).linearResponseTime(Duration.ofMillis(60), capacity))
-                        .until(Duration.ofSeconds(3), "fast 500s")
-                        .handler(h -> h.response(500).linearResponseTime(Duration.ofMillis(10), capacity))
+                        .handler(h -> h.response(200).responseTime(Duration.ofMillis(120)))
+                        .build(),
+                SimulationServer.builder()
+                        .serverName("fast_503s_then_revert")
+                        .simulation(simulation)
+                        .handler(h -> h.response(200).responseTime(Duration.ofMillis(120)))
+                        .until(Duration.ofSeconds(3), "fast 503s")
+                        .handler(h -> h.response(503).responseTime(Duration.ofNanos(10)))
```

Comment:
I think we want one sever that always 503s.

Reply:
I actually think seeing the transitions are also very useful, and representative of the real world. You get a solid 7 seconds of pure brokenness in the middle

Reply:
That's fair, good point :+1: